crates/noirc_frontend/src/hir/def_map/aztec_helper.rs

```diff
@@ -0,0 +1,202 @@
+use noirc_errors::Span;
+
+use crate::{
+    token::Attribute, CallExpression, Distinctness, Expression, ExpressionKind, FunctionReturnType,
+    Ident, LetStatement, MethodCallExpression, NoirFunction, ParsedModule, Pattern, Statement,
+    UnresolvedType, Visibility,
+};
+
+pub(crate) fn aztec_contracts_macros(ast: &mut ParsedModule) {
+    // Usage -> mut ast -> AztecLib.transform(&mut ast)
+
+    // TODO: rm
+    // loop over all of the functions and print their attributes, to check what attributes look like and if they
+    // can be added at will
+    // NOTE: only one attribute can be applied per function at the moment
```

Comment:
Can we remove the TODO? If this is referencing the eventual desire to remove this in favor of a more general macro system, we should track that with github issues rather than within the code.